src/components/VFlow/VFlow.test.tsx

```diff
@@ -25,3 +25,18 @@ it('should accept "style" prop', () => {
   )
   expect(container).toMatchSnapshot()
 })
+
+it('should render wrappers only for valid children', () => {
+  const { container } = render(
+    <VFlow>
+      {null}
+      {undefined}
+      {true}
+      {false}
+      {0}
+      {1}
+      {''}
+    </VFlow>
+  )
+  expect(container.querySelectorAll('div')[0].childNodes.length).toEqual(3)
```

Comment:
what do you think about adding a "expect line" to check the snapshot too?

Reply:
It's a good idea, I'll do it :+1: